.github/workflows/hermes.yml

```
@@ -0,0 +1,96 @@
+          persist-credentials: false
+          submodules: recursive
+
+      - name: Install Hermes Agent
+        run: |
+          curl -fsSL https://hermes-agent.nousresearch.com/install.sh | bash
+          echo "$HOME/.local/bin" >> "$GITHUB_PATH"
+
+      - name: Configure Hermes (non-secret settings only)
+        env:
+          HERMES_HOME: ${{ secrets.HERMES_HOME || '$HOME/.hermes' }}
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC1009:info:7:3: The mentioned syntax error was in this simple command

(shellcheck)